wdl-format/tests/format/seaseq-case/source.formatted.wdl

```diff
@@ -0,0 +1,939 @@
+            string_sra,
+        ])
+        scatter (eachsra in s_sraid) {
+            call sra.fastqdump { input:
+                sra_id = eachsra,
+                cloud = false,
+            }
+        }  # end scatter each sra
+
+        Array[File] sample_srafile = flatten(fastqdump.fastqfile)
+    }  # end if sample_sraid
+
+    # Generating INDEX files
+    #1. Bowtie INDEX files if not provided
+    if (!defined(bowtie_index)) {
+        # create bowtie index when not provided
+        call bowtie.index as bowtie_idx { input: reference = reference }
+    }
+    #2. Make sure indexes are six else build indexes
+    if (defined(bowtie_index)) {
+        # check total number of bowtie indexes provided
+        Array[String] string_bowtie_index = [
+            1,
+        ]  #buffer to allow for bowtie_index optionality
+        Array[File] int_bowtie_index = select_first([
+            bowtie_index,
+            string_bowtie_index,
+        ])
+        if (length(int_bowtie_index) != 6) {
+            # create bowtie index if 6 index files aren't provided
+            call bowtie.index as bowtie_idx_2 { input: reference = reference }
+        }
+    }
+    Array[File] actual_bowtie_index = select_first([
+        bowtie_idx_2.bowtie_indexes,
+        bowtie_idx.bowtie_indexes,
+        bowtie_index,
+    ])
+
+    # Spike-in DNA
+    #3. Bowtie INDEX files if not provided
+    String string_spikein = "1"
+    Array[String] string_spikein_buffer = [
+        1,
+    ]
+    if (!defined(spikein_bowtie_index) && defined(spikein_reference)) {
+        # create bowtie index on spikein genome
+        call bowtie.index as spikein_bowtie_idx { input: reference = select_first([
+            spikein_reference,
+            string_spikein,
+        ]) }
+    }
+
+    #4. Make sure indexes are six else build indexes for Spike-in DNA
+    if (defined(spikein_bowtie_index)) {
+        # check total number of bowtie indexes provided
+        Array[File] int_spikein_bowtie_index = select_first([
+            spikein_bowtie_index,
+            string_spikein_buffer,
+        ])
+        if (length(int_spikein_bowtie_index) != 6) {
+            # create bowtie index if 6 index files aren't provided
+            call bowtie.index as spikein_bowtie_idx_2 { input: reference = select_first([
+                spikein_reference,
+                string_spikein,
+            ]) }
+        }
+    }
+    Array[File] actual_spikein_bowtie_index = select_first([
+        spikein_bowtie_idx_2.bowtie_indexes,
+        spikein_bowtie_idx.bowtie_indexes,
+        spikein_bowtie_index,
+        string_spikein_buffer,
+    ])
+
+    # FASTA faidx and chromsizes and effective genome size
+    call samtools.faidx as samtools_faidx {
+        # create FASTA index and chrom sizes files
+        input: reference = reference }
+    call util.effective_genome_size as egs {
+        # effective genome size for FASTA
+        input: reference = reference }
```

Comment:
Also the closing bracket should be on a newline.

Reply:
It looks like this happens when there is an interrupting comment in an `input` block that would otherwise be a single line.

Reply:
Agree.